modules/ossm-migrating-a-multitenant-deployment.adoc

```diff
@@ -0,0 +1,179 @@
+== Migrating a multitenant deployment
+
+.Prerequisites
+
+* You have deployed {ocp-product-title} 4.14 or later.
+* You are logged in to the {ocp-product-title} web console as a user with the cluster-admin role.
+* You have completed the premigration checklists.
+* You have the {SMProduct} {SMv2Version} Operator installed.
+* You have the {SMProduct} 3 Operator installed.
+* You have `IstioCNI` installed.
+* You have `istioctl` installed.
+* You are running a `MultiTenant` `ServiceMeshControlPlane`.
+
+.Procedure
+
+The `bookinfo` example application is being used for demonstration purposes with a minimal example for the `Istio` resource. For more information on configuration differences between the {SMProduct} 2 `ServiceMeshControlPlane` resource and the {SMProduct} 3 `Istio` resource, see <insert whatever this file gets called>. You can follow these same steps with your own workloads.
+
+. Create your `Istio` resource.
++
+.Example `Istio` resource
++
+[source,yaml]
+----
+   apiVersion: sailoperator.io/v1
+   kind: Istio
+   metadata:
+     name: istio-tenant-a
+   spec:
+     namespace: istio-system-tenant-a <1>
+     version: v1.24.3
+     values:
+       meshConfig:
+         discoverySelectors: <2>
+           - matchLabels:
+               tenant: tenant-a
+   #     extensionProviders:  <3>
+   #       - name: prometheus
+   #         prometheus: {}
+   #       - name: otel
+   #         opentelemetry:
+   #           port: 4317
+   #           service: otel-collector.opentelemetrycollector-3.svc.cluster.local
+----
++
+<1> The `spec.namespace` field in your `Istio` resource must be the **same** namespace as your `ServiceMeshControlPlane`. If you set the `spec.namespace` field in your `Istio` resource to a different namespace than your `ServiceMeshControlPlane`, the migration will not work properly.
+<2> controlplanes by default watch the entire cluster and when managing multiple controlplanes on a single cluster, you must narrow the scope of each controlplane by setting `discoverySelectors`. In this example, we use the label `tenant` but you can use any label or combination of labels that you choose.
+<3> If you are migrating metrics and tracing, uncomment and update the `extensionProviders` fields according to your tracing and metrics configurations.
+
+. Add your `tenant` label to each one of your dataplane namespaces by running the following command for each dataplane namespace:
++
+[source,terminal]
+----
+$ oc label ns bookinfo tenant=tenant-a
+----
++
+[NOTE]
+====
+With {SMProduct} 2.6, namespaces were enrolled into the mesh by adding them to the `Service MeshMemberRoll` resource. In {SMProduct} 3, you must label each one of your dataplane namespaces to match your `discoverySelectors`.
+====
+
+=== Migrating workloads
+
+Now you can migrate your workloads from the {SMProduct} 2.6 controlplane to the {SMproduct} 3.0 controlplane.
+
+[NOTE]
+====
+You can migrate workloads and gateways separately.
```

Comment:
Maybe also add "and in any order" so it's more clear that you don't need to upgrade workloads _before_ you upgrade gateways.
```suggestion
You can migrate workloads and gateways separately.
```